exchange/bidder.go

```diff
@@ -321,6 +324,7 @@ func (bidder *bidderAdapter) doRequest(ctx context.Context, req *adapters.Reques
 	}
 	httpReq.Header = req.Headers
 
+	ctx = bidder.AddClientTrace(ctx)
```

Comment:
Should we add a feature flag around this option? Just in case it doesn't work as expected for some reason in prod or if a host would like to disable the feature to save a little bit in memory usage. If so, the prometheus preload should also only preload the metrics if the feature flag is enabled.

Reply:
Added. Let me know your thoughts with the addition